### ISSUE TITLE
Bot travels to the origin pose in compute_cartesian_path() while using MoveIt Python API

### ISSUE THREAD

**Issue by topguns837** [moveit/moveit#3657]
### Description

After the new changes in this PR(https://github.com/moveit/moveit/pull/3618), I am observing some weird motion while using the `compute_cartesian_path()`, where the bot travels back to the origin pose after reaching each of the waypoint pose

### Your environment
* ROS Distro: Noetic
* OS Version: Ubuntu 20.04
* Build: Binary
* Release Version: 1.1.15-1focal.20240914.082856

### Expected behaviour
The bot should travel from one waypoint to the next and complete the whole sequence of motion

https://github.com/user-attachments/assets/4baa3c7d-880f-476b-8ee9-616f7b90594a

### Actual behaviour
The bot goes to the first waypoint, then returns to its pose before starting the cartesian plan execution, then goes to the second waypoint, returns back etc.

https://github.com/user-attachments/assets/00f899ca-aee0-4f8e-9f8c-858f82205dab

**Comment by rhaschke** [comment]
This is a duplicate of #3651, which was already fixed via #3652. A new release was triggered but waits for sync.
Meanwhile, consider using the debian packages from the [ROS `testing` repo](http://wiki.ros.org/TestingRepository) or build from source.
